src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static add(animal) {
+    Animal.alive.push(animal);
+  }
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
```

Comment:
all animals should have health of 100 by default
```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;
```